support/oidc-discovery-provider/README.md

```diff
@@ -60,10 +60,6 @@ The configuration file is **required** by the provider. It contains
 
 #### Considerations for Windows platforms
 
-[1]: One of `acme` or `listen_named_pipe_name` must be defined.
```

Comment:
Looks like these lines were removed and not re-added? Was that intentional?

Reply:
no.... I'll revert this one

Reply:
my bad... It was because duplicated [1] and [3] I just restored